src/main/java/io/github/moulberry/notenoughupdates/profileviewer/SkyblockProfiles.java

```diff
@@ -942,6 +942,9 @@ public Map<String, ProfileViewer.Level> getLevelingInfo() {
 				if (skillName.equals("farming")) {
 					maxLevel += Utils.getElementAsInt(Utils.getElement(profileJson, "jacobs_contest.perks.farming_level_cap"), 0);
 				}
+				if (skillName.equals("taming")) {
+					maxLevel += Utils.getElement(profileJson, "pets_data.pet_care.pet_types_sacrificed").getAsJsonArray().size();
```

Comment:
shouldnt we provide a default here as fallback in case that element does not exist?